Resources/config/services.xml

```diff
@@ -33,7 +33,9 @@
         </service>
 
         <service id="presta_sitemap.dump_command" class="Presta\SitemapBundle\Command\DumpSitemapsCommand" public="true">
-            <tag name="console.command"/>
+            <argument type="service" id="router" />
+            <argument type="service" id="Presta\SitemapBundle\Service\DumperInterface" />
```

Comment:
Please inject the same service that was retrieved in the first place : `presta_sitemap.dumper`

Reply:
What it if alias my own service to interface?

Reply:
`dumper` bundle config entry needs to take precedence over your alias, otherwise it's useless. So @yann-eugone is right.